### ISSUE TITLE
PHP error when reading pihole-FTL.conf

### ISSUE THREAD

**Issue by Olivier2561** [pi-hole/FTL#391]
I copied pihole-FTL.conf from /etc/.pihole/advanced/Templates/pihole-FTL.conf to /etc/pihole and when restarting the system from the web interface, I get the error:

```
Error

There was a problem applying your settings.
Debugging information:
PHP error (2): syntax error, unexpected '(' in /etc/pihole/pihole-FTL.conf on line 33 in /var/www/html/admin/settings.php:15
```

when checking the line 33, it reads:

```
## How long should queries be stored in the database (days) ?
```

and sure enough, when removing the parenthesis, the error disappears (jumps to the next line with parenthesis).

I am using a stock pi-hole 4.0 on latest raspbian.

**Comment by DL6ER** [comment]
This happens because PHP doesn't accept `#` as a valid comment character any more. It is a PHP "feature". We can change the file to use PHP-style comments (starting in `;`) as FTL can understand this as well. Thanks for pointing this out!

**Comment by DL6ER** [comment]
This bug will be fixed by https://github.com/pi-hole/pi-hole/pull/2431 which is milestoned to be included in Pi-hole v4.1.

You can go ahead and replace all `#` by `;` in your `pihole-FTL.conf` file to remove the error.
